docs/src/pages/[platform]/connected-components/authenticator/customization/customization.i18n.react-native.mdx

```diff
@@ -23,7 +23,7 @@ These [translations](https://github.com/aws-amplify/amplify-ui/blob/main/package
 
 <Fragment>{({ platform }) => import(`./i18n/i18n.${platform}.mdx`)}</Fragment>
 
-The list of available keys are available [here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts).
+The list of available keys are available [here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/dictionaries/authenticator/en.ts).
```

Comment:
How does making this change prevent the same issue from occurring in the future?

Reply:
After chatting offline, the main concern is that linking to source code is fragile.

Reply:
Long term fix could be to embed the text and to snapshot test all the docs pages? This is a potential problem with any fragment, right?